humanitec-resource-defs/redis/basic/main.tf

```diff
@@ -78,11 +77,13 @@ service.yaml:
       ports:
       - name: tcp-redis
         port: {{ .init.port }}
-        targetPort: {{ .init.port }}
+        targetPort: tcp-redis
 EOL
         outputs   = <<EOL
 host: {{ .init.name }}
 port: {{ .init.port }}
+username: ""
+password: ""
```

Comment:
If those properties aren't present, the placeholders aren't resolved by Humanitec. Add them as empty strings.